cellbrowser_tools/dataHandoffUtils.py

```diff
@@ -145,6 +209,11 @@ def collect_data_rows(fovids=None, raw_only=False, max_rows=None):
 
     print("GOT DATA HANDOFF")
 
+    # Merge Aligned and Source read path columns
+    df_data_handoff[DataField.SourceReadPath] = df_data_handoff[
+        DataField.AlignedImageReadPath
+    ].combine_first(df_data_handoff[DataField.SourceReadPath])
+
```

Comment:
I feel like I have seen this snippet like 4 times now.... maybe I am following the code incorrectly but I don't really see why it's needed so many times.